plugins/modules/sonic_fbs_classifiers.py

```diff
@@ -0,0 +1,472 @@
+      match_hdr_fields:
+        description:
+          - Match header fields configuration
+          - I(ipv4) and I(ipv6) are mutually exclusive.
+        type: dict
+        suboptions:
+          ip:
+            description:
+              - IP field configuration
+            type: dict
+            suboptions:
+              dscp:
+                description:
+                  - Value of diffserv code point, range 0-63
+                type: int
+              protocol:
+                description:
+                  - IP protocol
+                type: str
+                choices: ['auth', 'gre', 'icmp', 'icmpv6', 'igmp', 'l2tp', 'pim', 'rsvp', 'tcp', 'udp']
+          ipv4:
+            description:
+              - IPv4 field configuration
+            type: dict
+            suboptions:
+              source_address:
+                description:
+                  - Source IPv4 address prefix
+                type: str
+              destination_address:
+                description:
+                  - Destination IPv4 address prefix
+                type: str
+          ipv6:
+            description:
+              - IPv6 field configuration
+            type: dict
+            suboptions:
+              source_address:
+                description:
+                  - Source IPv6 address prefix
+                type: str
+              destination_address:
+                description:
+                  - Destination IPv6 address prefix
+                type: str
+          l2:
+            description:
+              - Ethernet field configuration
+            type: dict
+            suboptions:
+              source_mac:
+                description:
+                  - Source MAC address
+                type: str
+              source_mac_mask:
+                description:
+                  - Source MAC address mask
+                type: str
+              destination_mac:
+                description:
+                  - Destination MAC address
+                type: str
+              destination_mac_mask:
+                description:
+                  - Destination MAC address mask
+                type: str
+              dei:
+                description:
+                  - Drop eligible indicator, range 0-1
+                type: int
+              ethertype:
+                description:
+                  - Ethertype field to match in ethernet packets
+                type: str
+                choices: ['arp', 'ipv4', 'ipv6', 'lldp', 'mpls', 'roce', 'vlan']
+              pcp:
+                description:
+                  - Priority code point, range 0-7
+                type: int
+              vlanid:
+                description:
+                  - VLAN ID, range 1-4094
+                type: int
+          transport:
+            description:
+              - Transport field configuration
+            type: dict
+            suboptions:
+              source_port:
```

Comment:
Please include in the port descriptions a description of the range delimiter used for this resource module.

It would be more consistent with other modules in our collection to use '-' as the range delimiter, but it's okay to use ".." instead if it is described here. Even if the '-' notation is used, it would be worthwhile to state that in the description.